python/ql/src/experimental/semmle/python/frameworks/MySQLdb.qll

```diff
@@ -0,0 +1,23 @@
+private import python
+private import experimental.dataflow.DataFlow
+private import experimental.dataflow.RemoteFlowSources
+private import experimental.semmle.python.Concepts
+private import PEP249
+
+// ---------------------------------------------------------------------------
+// MySQLdb
+// ---------------------------------------------------------------------------
+/** Gets a reference to the `MySQLdb` module. */
+private DataFlow::Node moduleMySQLdb(DataFlow::TypeTracker t) {
```

Comment:
I see that we're getting into trouble with the naming scheme since we can't let the predicate start with an upper-case letter :neutral_face: 

I think this is an OK solution (and certainly can't come up with anything better)